plugins/patterns/schema/system-description-patterns.schema-v10.json

```diff
@@ -7,7 +7,22 @@
   ],
   "properties": {
     "_attributes": {
-      "type": "object"
+      "type": "object",
+      "required": [
+        "patterns_system"
+      ],
+      "properties": {
+        "patterns_system": {
+          "oneOf": [
+            {
+              "enum": ["zypper"]
```

Comment:
I basically copied the concept from the v10 packages schema (there's a lot more duplication in there as it doesn't use ```oneOf```) but I can also change it I guess.